testing/fix-macros.h

```diff
@@ -0,0 +1,26 @@
+/* -*- mode: c++ -*-
+ * Copyright (C) 2020  Eric Paniagua (epaniagua@google.com)
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// No `#pragma once` since these undefs need to have every time a Kaleidoscope/
+// Arduino header is included before non-Kaleidoscope/Arduino header.
+// #pragma once
+
```

Comment:
Ideally, this would have a comment about it being specifically because Arduino and gtest (is that right?) have naming conflicts.